### DISCUSSION TITLE
Electrs permanently 'waiting for bitcoin node to finish syncing...'?

### DISCUSSION THREAD

**Discussion by AlekPiasecki** [romanz/electrs#921] [General]
Unclear why I am getting this but when opening electrs it keeps telling me it is waiting for the bitcoin node to finish syncing and effectively remains in this state. I can confirm that the bitcoin node is 100% synced and up to date, but the electrs progress does not continue. I have tried restating both. I also removed electrs, restarted the node, downloaded electrs and still no improvement. What is going on?

**Comment by romanz** [comment]
Could you please attach electrs logs?

**Reply by perfectlycalm** [reply]
Where do you fetch the log from?

**Reply by romanz** [reply]
Usually the log is written to stderr (if electrs is run from command-line, see [doc/usage.md](https://github.com/romanz/electrs/blob/master/doc/usage.md)).
How are you running electrs?

**Reply by perfectlycalm** [reply]
I'm running it on Umbrel.  I don't really know how to use the console.  I can get the error logs from the Umbrel settings dashboard

**Reply by romanz** [reply]
CC: @getumbrel @lukechilds

**Comment by perfectlycalm** [comment]
This has been happening to me too as of yesterday.  I have to delete the Electrs app and reinstall it and wait for it to resynch to resolve
